### DISCUSSION TITLE
Instalation in Ubuntu

### DISCUSSION THREAD

**Discussion by MariaOlarteG** [thelfer/tfel#513] [General]
Hi. I'm a PhD student and I'm trying to install MFront in my computer with Linux system. When I do it using this documentations https://thelfer.github.io/tfel/web/install.html, I have some errors at the installation: 

1. Installing curl-8.6.0-74pd3fnz62gxyn32b37nixvkkuy64jb5 [30/43]
==> No binary for curl-8.6.0-74pd3fnz62gxyn32b37nixvkkuy64jb5 found: installing from source
==> Error: FetchError: All fetchers failed for spack-stage-curl-8.6.0-74pd3fnz62gxyn32b37nixvkkuy64jb5
==> Warning: Skipping build of cmake-3.27.9-iw6k654osuliqerhn6phpr3lkd4o2svr since curl-8.6.0-74pd3fnz62gxyn32b37nixvkkuy64jb5 failed
==> Warning: Skipping build of tfel-master-kfcwbti4p667dn4fjvxkodldddc4dkhy since cmake-3.27.9-iw6k654osuliqerhn6phpr3lkd4o2svr failed

2. ==> Installing boost-1.84.0-v4pcwmtrva5chsdic42zo36ixlstgnza [41/43]
==> No binary for boost-1.84.0-v4pcwmtrva5chsdic42zo36ixlstgnza found: installing from source
==> Fetching https://boostorg.jfrog.io/artifactory/main/release/1.84.0/source/boost_1_84_0.tar.bz2
==> Applied patch /home/mcolarte/spack/var/spack/repos/builtin/packages/boost/python_jam-1_77.patch
==> Applied patch /home/mcolarte/spack/var/spack/repos/builtin/packages/boost/bootstrap-compiler.patch
==> Ran patch() for boost
==> boost: Executing phase: 'install'
==> boost: Successfully installed boost-1.84.0-v4pcwmtrva5chsdic42zo36ixlstgnza
  Stage: 32.58s.  Install: 30.18s.  Post-install: 2.04s.  Total: 1m 4.91s
[+] /home/mcolarte/spack/opt/spack/linux-ubuntu22.04-skylake/gcc-11.4.0/boost-1.84.0-v4pcwmtrva5chsdic42zo36ixlstgnza
==> Error: tfel-master-kfcwbti4p667dn4fjvxkodldddc4dkhy: Package was not installed
==> Error: Installation request failed.  Refer to reported errors for failing package(s). 

So I can't proceed with "spack load tfel@master". 

I tried with the "Quick instalation procedure", but in the moment when I execute ccmake ../tfel, I dont find: 
  TFEL_APPEND_VERSION           ON
  enable-numpy-support          ON
  enable-python                 ON
  enable-python-bindings        ON

So I cant configure. 

I hope you can help me with this... 

Regards, 

Maria.

**Comment by thelfer** [comment]
Dear Maria,

`master` is currently being ported to `C++20`. It does require a very recent compiler. Maybe could you use one of the released version or one of `rliv` branches. For example:

```
spack install tfel@4.2.0
```
